### ISSUE TITLE
Semantically incorrect Cyclus Testing heading

### ISSUE THREAD

**Issue by scopatz** [cyclus/cyclus.github.com#174]
The heading that reads on the cyclus user docs:

```
Cyclus Testing
    The Cyclus kernel includes some archetype for testing the basic functionality of the kenel.

    Cyclus Archetypes
        :agents:Source
        :agents:Sink
        :agents:NullRegion
        :agents:NullInst
        :agents:KFacility
        :agents:Predator
        :agents:Prey

```

is not right.  Not all of these archetypes are for testing and I would argue that some, such as the Nulls are needed for day-to-day usage.

**Comment by rwcarlsen** [comment]
Yes - this distinction has come up here (madison) in discussion before.  The Nulls are really the ones that don't fit solely under the testing umbrella. @gonuke was working on refactoring this page, and we settled on the current state as an improvement, but with the caveat that there were still some changes that probably need to be made.  Thanks for posting the issue @scopatz.
